.github/CODEOWNERS

```diff
@@ -87,9 +88,6 @@
 /packages/expo-web-browser                      @bbarthec
 /packages/expo-yarn-workspaces                  @ide
 /packages/expo                                  @ide @tsapeta
-/packages/expo/cli                              @EvanBacon @bycedric
```

Comment:
Add back a rule for `/packages/@expo/cli`

Reply:
It's at the top of the file https://github.com/expo/expo/pull/16717/files#diff-3d36a1bf06148bc6ba1ce2ed3d19de32ea708d955fed212c0d27c536f0bd4da7R1